utils.go

```diff
@@ -512,6 +512,20 @@ func isAmzHeader(headerKey string) bool {
 	return strings.HasPrefix(key, "x-amz-meta-") || strings.HasPrefix(key, "x-amz-grant-") || key == "x-amz-acl" || isSSEHeader(headerKey) || strings.HasPrefix(key, "x-amz-checksum-")
 }
 
+var supportedReplicationHeaders = map[string]bool{
+	"x-minio-replication-server-side-encryption-sealed-key":     true,
+	"x-minio-replication-inline-data":                           true,
```

Comment:
```suggestion
	"x-minio-replication-inline-data":                           true,
```

is this needed? @shtripat

Reply:
may not be. will try removing and testing.

Reply:
removed now as not needed.